src/main/java/com/vishva/Payapp/exchange/CreditRequest.java

```diff
@@ -12,7 +15,11 @@
 public class CreditRequest {
 
     @NotNull
+    @Pattern(regexp = "[0-9]+")
```

Comment:
I tried to test your code but i don't think your validation works. Remember abhinav was getting the same issue then he mentioned that with spring version > 2 need one more dependency to make validation works. Please check it and make it running.

Reply:
I think the next one, min and max was not working. For accId when we enter something other than the digits like e.g. abc or something then it throws that pls enter correct accId message, isn't it what we are expecting here?